to_string.hpp

```diff
@@ -9,6 +9,9 @@
 
 #include <type_traits>
 
+constexpr char digits[] = "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZ";
```

Comment:
`inline constexpr `

Reply:
@Neargye why `inline constexpr` when `constexpr` itself is implicitly inline?

Reply:
​constexpr specifier implies  inline for static data members as well as functions, but not for global variable.